pageserver/src/repository/rocksdb.rs

```diff
@@ -480,9 +488,53 @@ impl RocksTimeline {
         Ok(0)
     }
 
+    ///
+    /// Drop relations with all its forks or non-relational file
+    ///
+    fn drop(&self, tag: BufferTag) -> Result<()> {
```

Comment:
Please don't use the function name `drop`.

In rust, `drop()` is expected to mean the `Drop` trait, for discarding/destructing objects.

Reply:
Done